packages/engine/src/worker/runner/python/hash_util.py

```diff
@@ -1,14 +1,17 @@
+from copy import deepcopy
 from json import loads
+
 import pyarrow as pa
 from pyarrow.types import is_primitive
+
 from wrappers import np_force_writable
 
 
 def json_deepcopy(x):
     return deepcopy(x)
 
 
-def load_shallow(vector):
+def load_shallow(vector, is_nullable, is_any):  # TODO: Change arguments after upgrading Arrow.
```

Comment:
Is there a reason we have `is_nullable` and `is_any` here? Is it just to match the signature of `load_full`? Or is it to do with the TODO

Reply:
Yeah just to match `load_full`. It might be possible to go back to just `vector` after upgrading Arrow.

Reply:
Unless there's a need to match the load_full signature (e.g. if we're collecting both types of functions in an iterable or something) I think we should remove the unused arguments

Reply:
It's necessary because they can both be used as custom loaders (e.g. if you want to fully load a private field or shallow-load an agent-scoped field)